### PR TITLE
Added source inception dates

### LINKED ISSUE #121: Indicate when a source was added
Similar to how we indicate a source was deprecated (for those bad properties that should have been in the other repo).

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #121.

Added `edtf:inception` dates for each of the source json files in this repo, ran makefile, updated README, and template file.

Tagging @nvkelso for review, all files are changing.

### CODE REVIEW

File: bin/docs.py [hunk 1]
```diff
@@ -68,6 +68,10 @@
             if not details['edtf:deprecated'] == 'uuuu':
                 docs.write("* %s %s.\n" % ('This source was deprecated on', details['edtf:deprecated']))
 
+        #call out add date
+        if details.get('edtf:inception'):
+            docs.write("* %s %s.\n" % ('This source was added on', details['edtf:inception']))
```
Comment (nvkelso) [inline bin/docs.py]: Please instead say:

- added: `YYYY-MM-DD`

Like:

`docs.write("* %s: `%s`\n" % ('added', details['edtf:inception']))`

Comment (nvkelso) [reply bin/docs.py]: Also update deprecated to this same format, please.

Comment (nvkelso) [reply bin/docs.py]: Say added, then deprecated.

**Comment by stepps00** [comment]
Most recent commits take care of comments.

**Pull request merged**